pkg/app/mutate_app.go

```diff
@@ -76,10 +76,12 @@ func (m *Mutator) Mutate(request *v1beta1.AdmissionRequest) ([]mutator.PatchOper
 
 	m.logger.Debugf(ctx, "mutating app %#q in namespace %#q", appNewCR.Name, appNewCR.Namespace)
 
-	// We check the deletion timestamp because app CRs may be deleted by
-	// deleting the namespace they belong to.
 	if request.Operation == v1beta1.Update && !appNewCR.DeletionTimestamp.IsZero() {
-		m.logger.Debugf(ctx, "admitted deletion of app %#q in namespace %#q", appNewCR.Name, appNewCR.Namespace)
+		m.logger.Debugf(ctx, "skipping mutation for UPDATE operation of app %#q in namespace %#q with non-zero deletion timestamp", appNewCR.Name, appNewCR.Namespace)
+		return nil, nil
+	}
+	if request.Operation == v1beta1.Connect {
```

Comment:
Ignoring connect.

Reply:
Wow nice catch. I didn't know connect operations were a thing.

The new message is also clearer about the deletion timestamp. 👍

Reply:
It is but it won't be dispatched as [we don't subscribe to it](https://github.com/giantswarm/app-admission-controller/blob/c614852d5787a6a7cb5141dd56ab190eb7181224/helm/app-admission-controller/templates/webhook.yaml#L26-L28). I may as well remove it.